### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN008.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#54] [General]
# https://blog.3place.cn/3place/lmyn/LMYN008.html

https://blog.3place.cn/3place/lmyn/LMYN008.html

**Comment by picklecai** [comment]
舜有臣五人而天下治。武王曰：“予有乱臣十人。”

其中说到“昂昂公侯，实天诞育，八元斯九，五臣兹六”，对人臣的赞美，这些话都是顶配。老天爷为了让世界更美好，刻意创造了一个贾充。尧舜时代的贤臣有所谓“八元”，加上贾充就有九个了；舜治理天下靠“五臣”，有了贾充从此就是六个。——作者：潘岳

潘岳就是潘江陆海的潘江，貌比潘安的潘安。

**Comment by picklecai** [comment]
要想说话，总是先满面含笑，

这是大臣的礼仪。

**Comment by picklecai** [comment]
他们决不会把谎话来说，

这是大臣的诚。

**Comment by picklecai** [comment]
怎么说话就怎么行事。 

这是大臣的知行合一。

**Comment by picklecai** [comment]
对自己方面，对别人方面，
他们没有不知道的事情。
他们已经做或准备做的事，
都是通过密探去完成。 

密探不是重点，重点是他们知己知彼，对王国内的事情了如指掌，以便一切可控。

基本认知是：了解是管理的前提。
这个时代，可能还没有如果不知道、如果超纲，该怎么解题的总结。所以，最高赞誉就是：啥都知道。

密探的意思是：他们了解情况的手段，是没有打扰到人民日常生活的。所以依赖于密探，也是赞誉。

不扰民，就是这个时代的价值观。不用今天的眼光评判的话，这个标准一点儿也不落后。今天仍然有许多时候，不如这个。

**Comment by picklecai** [comment]
即使自己的儿子犯了罪，
也会按照情节加以惩办。

王子犯法，与庶民同罪

**Comment by picklecai** [comment]
他们致力于积累财富，
也努力去募集军队，
连坏人敌人都不杀害，
只要他们不去犯罪。 

朱元璋：广积粮，高筑墙，缓称王。

**Comment by picklecai** [comment]
深通达磨。
深通经典。
深通治术。

对罗摩的要求是深通达磨，要有脑子，有战略，有高度。
对大臣的要求是深通经典和治术，要知道得多，并能解决实际问题。

**Comment by picklecai** [comment]
不杀害婆罗门和刹帝利。

虽然以出身决定在今天看上去那么不可思议，但是不以人的自身表现而以其他的属性来决定他是否值得，这一点，直到今天也很难完全戒除。

**Comment by picklecai** [comment]
在善人和专心一志的人中间，
在所有的十分睿智的人中间，
在城市里，还有在乡村中，
没有任何一个人会说谎言。 

没有任何一个坏人，
会对别人妻子欢心，
整个国家和整个城，
都是宁静而又安稳。

所有的人穿住都很好，
所有的人都守法知义，
为了使国王快乐幸福，
他们警惕地遵守仪礼。

这是周公和孔子的理想。

**Comment by picklecai** [comment]
他用密探当眼睛，
以法统治老百姓，

锦衣卫说你昨晚在家犯法了。

**Comment by picklecai** [comment]
这八人中，没有婆私吒。
所以，也许婆私吒是内宦？古印度有这样的身边的侍从或者啥吗？

**Reply by picklecai** [reply]
有两个亲密的大僧正，
婆私吒的缚摩提婆，
他们是仙人中的魁首。

应该是婆私吒和缚摩提婆。
原来婆私吒是大僧正。大概就是十车王身边的宗教人士。类似于大观园的妙玉，或者乾隆还是康熙电视剧里的一个和尚。
